fs/mmap/fs_munmap.c

```diff
@@ -158,7 +160,38 @@ static int file_munmap_(FAR void *start, size_t length, bool kernel)
   nxmutex_unlock(&g_rammaps.lock);
   return ret;
 #else
+
+#ifdef CONFIG_MM_MAP
```

Comment:
let's remove g_rammaps and reuse tg_mm_map, to avoid the code duplication

Reply:
Didn't do this yet, I will look into this one soon